qa/rpc-tests/exodus_walletrecovery.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+import os
+from shutil import rmtree
+from test_framework.test_framework import ExodusTestFramework
+from test_framework.util import (
+    assert_equal,
+    connect_nodes_bi,
+    start_node,
+    stop_node,
+    sync_blocks)
+
+class ExodusWalletRecoveryTest(ExodusTestFramework):
+    def get_datadir(self, node = 0):
```

Comment:
We should not provide the default node due to when invoking without specify the value it will really confusing.